client/actions/group.js

```diff
@@ -0,0 +1,79 @@
+import axios from 'axios';
+
+/**
+ * @class GroupActions
+ */
+export default class GroupActions {
+  /**
+  * Request to the API to create a group
+  *
+  * @static
+  * @param {String} groupName The name of the group to be created
+  * @param {String} description The description of the group to be created
+  * @param {Array} groupMembers The members of the group to be created
+  * @param {String} token JWToken to access the endpoint
+  * @returns {Object} dispatch object
+  * @memberof GroupActions
+  */
+  static createGroup(groupName, description, groupMembers, token) {
+    return (dispatch) => {
+      dispatch({ type: 'CREATE_GROUP_BEGINS' });
```

Comment:
It better you use constant as your action type, in that way you will avoid spelling errors that can cause bug. You can create a file and define all your constants then export it e.g `export const CREATE_GROUP_BEGINS = "CREATE_GROUP_BEGINS"`, then import and use it anywhere you need it